cmd/influxd/upgrade/upgrade_test.go

```diff
@@ -236,6 +237,9 @@ func TestUpgradeRealDB(t *testing.T) {
 	v.SetConfigFile(configPath)
 	require.NoError(t, v.ReadInConfig())
 	lOpts := launcher.NewOpts(v)
+	// We need to specify the path to sqlite here to prevent errors from trying to write to the database
+	// concurrently by parallel tests.
```

Comment:
What sorts of errors were you seeing? I wouldn't expect anything in the `upgrade` command to interact with sqlite, and wonder why we haven't hit issues w/ the bolt DB

Reply:
I actually think this was a leftover from when I was trying some different things out before. I know for sure I was getting some kind of errors because the database was locked, but I was able to remove it for now without issue. So we should be good to go without this!